### PR TITLE
Remove redundant yank

### PR DESCRIPTION
**Pull request by Obaied**
Closes https://github.com/radare/radare2/issues/10937
Regressions tests PR: https://github.com/radare/radare2-regressions/pull/1425

First PR \o/

### CODE REVIEW

**Comment by Maijin** [comment]
Can you remove the deprecated messages and fix the tests (do a Pull request on radare2-regressions repository) - https://travis-ci.org/radare/radare2/jobs/413623119

**Comment by Obaied** [comment]
Ah alright. Will do!

File: libr/core/cmd_help.c [hunk 1]
```diff
@@ -425,12 +424,6 @@ static int cmd_help(void *data, const char *input) {
 		}
 		break;
 	case 'y': // "?y"
```
Comment (ret2libc) [inline libr/core/cmd_help.c]: I think this `case 'y'` should be completely removed

**Comment by ret2libc** [comment]
r2r PR: https://github.com/radare/radare2-regressions/pull/1425

**Comment by ret2libc** [comment]
@Obaied Please adjust your r2r PR to make travis pass. there are some tests which don't.

**Comment by Obaied** [comment]
@ret2libc Its passing locally when running `make all`. How can I run Travis with the specific R2R branch I pushed?

**Comment by ret2libc** [comment]
It's enough you push your changes to the r2r branch and then I can rerun travis (at that point travis will automatically get the updates on r2r and execute them)

**Comment by Obaied** [comment]
Does Travis just run `make all` or is there some other command I can reproduce locally? Since the R2R branch I made is passing those tests.

**Comment by ret2libc** [comment]
It just does `make all`

**Comment by ret2libc** [comment]
In particular I think you should fix the test in `new/db/cmd/cmd_yank`, as you can see from the travis log.

**Comment by Obaied** [comment]
I've made the amendment. My naive way of finding the issue was to run a string search, but I didn't realize the commands were b64 encoded :)
Things should work now. Thanks for the help and guidance. I look forward to contribute more and (hopefully) less error-prone than later :)

**Comment by ret2libc** [comment]
Thank you for contributing! And don't worry for the issues, some things may not be very clear ;)
I've restarted travis, which should get your new r2r branch. Also, I commented on the r2r PR

**Comment by Obaied** [comment]
I'm a bit confused. Any idea why Travis is still failing?

**Comment by ret2libc** [comment]
You probably have to rebase both r2 pr and r2r pr on top of masters because of recent changes

**Comment by Obaied** [comment]
Both rebased. Tests ran successfully.

**Comment by ret2libc** [comment]
Thanks for your contributions!
Keep contributing ;)

**Pull request merged**